### PR TITLE
[ADD] add account_prepayment from superman

### PR DESCRIPTION
**Pull request by xie8899**

### CODE REVIEW

File: account_prepayment/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (noahzaozao) [inline account_prepayment/__init__.py]: header

File: account_prepayment/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (noahzaozao) [inline account_prepayment/__openerp__.py]: header

File: account_prepayment/account.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (noahzaozao) [inline account_prepayment/account.py]: header

**Comment by xie8899** [comment]
@noahzaozao @elicoidal @seb-elico Done

**Comment by noahzaozao** [comment]
LGTM

File: account_prepayment/README.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Account Prepayment
+==================
+
+Fine tune Prepayment addon \n
```
Comment (elicoidal) [inline account_prepayment/README.rst]: remove \n
Add a blank line before the list

File: account_prepayment/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    'website': 'http://www.elico-corp.com',
+    'depends': [
+        'account',
+        'account_voucher',
+        'purchase'
+    ],
+    'data': [
+        'views/account_view.xml',
+    ],
+    'images': [],
```
Comment (elicoidal) [inline account_prepayment/__openerp__.py]: add the images

File: account_prepayment/README.rst [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+* Add Prepayment Account In Partner \n
+* Add purchase_order_id and Prepayment Account in payment.
+
```
Comment (elicoidal) [inline account_prepayment/README.rst:19]: add images

File: account_prepayment/models/account.py [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+        res['value'] = {'partner_id': po.partner_id.id, 'amount': amount}
+        return res
+
+    def onchange_prepayment_account(
+            self, cr, uid, ids, use_prepayment_account):
+        res = {}
+        line_cr_id = ""
+        if not use_prepayment_account:
+            line_cr_id = self.browse(cr, uid, fields)
+            print "\n\n\nll%s\n\n\n" % line_cr_id
```
Comment (elicoidal) [inline account_prepayment/models/account.py]: remove

File: account_prepayment/static/description/index.html [hunk 8]
```diff
@@ -0,0 +1,16 @@
+<section class="oe_container">
```
Comment (elicoidal) [inline account_prepayment/static/description/index.html]: remove this file (replaced by README)

File: account_prepayment/static/description/index.html [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+    <div class="oe_row">
+    <h2 class="oe_slogan">account_prepayment Module</h2> 
+    <h3 class="oe_slogan">Prepayment Account for Customers and <br>Usage of Prepayment Account for Payments in Purchase Order</h3>
+    <h4 class="oe_slogan"><a href="http://www.openerp.com.cn">By Elico Corp</a></h4>
+       <p>This module allows for creation of prepayment accounts for Customers. This module also allows for payment to Suppliers in Purchase Orders to be deducted from the Prepayment Accounts instead of just other normal payment accounts. </p>
```
Comment (elicoidal) [inline account_prepayment/static/description/index.html]: add this description in README

**Comment by noahzaozao** [comment]
@elicoidal @seb-elico @victormartinelicocorp shall we merge this PR ?

File: account_prepayment/README.rst [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+
+Contributors
+------------
+
+ * Andy Lu <andy.lu@elico-corp.com>
+
```
Comment (elicoidal) [inline account_prepayment/README.rst]: Add my self

**Review by elicoidal — APPROVED** [review]

**Comment by victor-ditay** [comment]
LGTM

**Comment by elicoidal** [comment]
LGTM

**Pull request merged**